File: pueblo_golf_tournament_mobile/lib/models/account.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:pueblo_golf_tournament_mobile/models/base-model.dart';
import 'package:pueblo_golf_tournament_mobile/utilities/enums.dart';
part 'account.g.dart';

@JsonSerializable()
class Account extends BaseModel {
  final String username;
  final String password;
  final double personId;
  final AccountTypeEnum accountType;

  Account(
      {required this.username,
      required this.password,
      required this.personId,
      required this.accountType,
      required super.id});

  factory Account.fromJson(Map<String, dynamic> json) =>
      _$AccountFromJson(json);

  Map<String, dynamic> toJson() => _$AccountToJson(this);
}

File: pueblo-golf-tournament-api/Entities/Account.cs
using pueblo_golf_tournament_api.Base;
using pueblo_golf_tournament_api.Utilities.Enums;

namespace pueblo_golf_tournament_api.Entities
{
    public class Account : BaseEntity
    {
        public string Username { get; set; } = "";
        public string Password { get; set; } = "";
        public long PersonId { get; set; }
        public AccountTypeEnum AccounType { get; set; }
        public virtual Person? Person { get; set; }
    }
}